src/menu/mod.rs

```diff
@@ -64,6 +64,7 @@ impl Plugin for MenuPlugin {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
It's not necessary, CI will pass anyways.
I'd remove it for now.

Reply:
Confusion like this is seriously tempting me to remove that allowance again 😄